mobile_app_connector/data/tile_type_data.xml

```diff
@@ -96,7 +96,7 @@ Thank You]]>
         <field name="view_order">5</field>
         <field name="default_model_id" ref="model_compassion_child"/>
         <field name="default_title">${ctx['objects'].name}</field>
-        <field name="default_body">Send a gift to your sponsored child</field>
+        <field name="default_body">Send a donation to your sponsored child</field>
```

Comment:
No, in this case we really invite him to do a gift so that's no problem to keep the word.